master/buildbot/steps/slave.py

```diff
@@ -19,6 +19,7 @@
 
 # pylint: disable=wildcard-import
 # pylint: disable=unused-wildcard-import
+from buildbot.steps.worker import WorkerBuildStep
```

Comment:
The trouble is that isort is changing the order of comments. I fixed it by just getting rid of the `*` import.